packages/front-b2b-link-rn/src/Front.tsx

```diff
@@ -55,10 +56,12 @@ const FrontFinance = (props: {
     ) {
       setShowWebView(false)
     }
-    if (type === 'brokerageAccountAccessToken') {
+    if (
+      (type === 'brokerageAccountAccessToken' && !isTransferLink) ||
```

Comment:
Web link does not ignore the `AccessTokenPayload` during TransferUX. I think both SDKs should be on the same page.

Reply:
fixed